operations/billingautomations.md

```diff
@@ -0,0 +1,707 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Billing automations
```

Comment:
thought: do partners know what is billing automation? is this name used in the UI, anywhere?

Reply:
Yup, we renamed routing rules to billing automations, it's used everywhere across the system and in the documentation.